tests/test_exceptions.py

```diff
@@ -60,15 +60,58 @@ def crash(self):
     with pytest.raises(ValueError) as excinfo:
         Something().crash()
 
-    assert "https://ploomber.io/community" in str(excinfo.value)
+    assert exceptions.get_community_link() in str(excinfo.value)
 
 
-def test_do_not_catch_other_errors():
+def test_modify_exceptions_type_error():
```

Comment:
We should still have this test but on a different error type (since now `TypeError` is in as well)